README.md

```diff
@@ -1,15 +1,65 @@
 
 ## Project prerequisites
-Make sure that you have the compatible versions of [Visual Studio 2022](https://visualstudio.microsoft.com/downloads/ ) and [.NET Core SDK 7.0](https://dotnet.microsoft.com/en-us/download/dotnet/7.0) or later version in your machine before starting to work on this project.
+Make sure that you have the compatible versions of [Visual Studio 2022](https://visualstudio.microsoft.com/downloads/) and [.NET Core SDK 7.0](https://dotnet.microsoft.com/en-us/download/dotnet/7.0) or later version in your machine before starting to work on this project.
 
 ## How to run this application
 To run this application, you need to first clone the `getting-started-with-the-dotnet-maui-maps` repository and then open it in Visual Studio 2022. Now, simply build and run your project to view the output.
+
+## Features and Benefits
+
+## Shape layer
+
+Render GeoJSON or shapefile data into custom maps or geographical area shapes on a shape layer for powerful data visualization. Customize the maps to the desired look using built-in options.
+
+## Tile layer
+
+An interactive tile layer allows you to load and customize map tiles from web map-tile services such as OpenStreetMaps, Bing Maps API, Google Maps Tile API, TomTom, Mapbox, Esri's ArcGIS. Use markers to denote a specific latitude and longitude in the tile layer as required. Zoom and pan the tile layer to navigate across the regions and take a closer look.
+
+## Data labels
+
+[Data labels](https://help.syncfusion.com/maui/maps/data-labels?utm_source=vs_marketplace&utm_medium=listing&utm_campaign=maui-maps-github-samples) in a shape layer provide basic information about shapes by displaying their names and more. If required, trim or hide the labels when they exceed the shape bounds.
+
+## Selection
+
+Highlight a shape to focus attention to a specific area. Use this functionality to navigate  pages and show detailed information about a selected area.
+
+## Zooming and panning
+
+Zoom in any layer for a closer look at a specific region by pinching, scrolling the mouse wheel, or track pad. Pan the map to navigate across regions. Zooming and panning can be enabled for both the shape layer and the tile layer.
+
+## Related Links
+
+[Learn More about .NET MAUI Maps](https://www.syncfusion.com/maui-controls/maui-maps?utm_source=vs_marketplace&utm_medium=listing&utm_campaign=maui-maps-github-samples)
+
+[Download Free Trial](https://www.syncfusion.com/downloads/maui/confirm?utm_source=vs_marketplace&utm_medium=listing&utm_campaign=maui-maps-github-samples)
```

Comment:
Use the below link for download free trial:
 https://www.syncfusion.com/downloads/maui

Reply:
updated